tests/test_input.py

```diff
@@ -80,20 +80,21 @@ def check_output_file(user_config_dict, simulation):
     """Check if metrics output is correctly written to file.
 
     Args:
+        simulation:
         user_config_dict: Dictionary with information to run user defined
```

Comment:
Minor doc string modification needed that there is no `simulation_config_dict`. And `simulation` is a string indicating name of simulated blend scene like "two_gal" or "group"

Reply:
thanks for the catch, corrected it in three functions